quarkchain/cluster/cluster_config.py

```diff
@@ -143,6 +143,11 @@ class ClusterConfig(BaseConfig):
     ENABLE_PRIVATE_JSON_RPC = True
     ENABLE_TRANSACTION_HISTORY = False
 
+    JSON_RPC_WS_PORT = 5000  # To be decided
+    PRIVATE_JSON_RPC_WS_PORT = 5050  # To be decided
```

Comment:
we probably don't need private jsonrpc port in websocket

Reply:
okay.